.eslintrc.js

```diff
@@ -19,19 +19,20 @@ module.exports = {
         'sourceType': 'module'
     },
     'rules': {
-        'new-cap': [
-            'error',
-            {
-                'capIsNew': false // Behave like JSHint's newcap.
-            }
-        ],
+        // 'new-cap': [
+        //     'error',
+        //     {
+        //         'capIsNew': false // Behave like JSHint's newcap.
+        //     }
+        // ],
         // While it is considered a best practice to avoid using methods on
         // console in JavaScript that is designed to be executed in the browser
         // and ESLint includes the rule among its set of recommended rules, (1)
         // the general practice is to strip such calls before pushing to
         // production and (2) we prefer to utilize console in lib-jitsi-meet
         // (and jitsi-meet).
         'no-console': 'off',
-        'semi': 'error'
+        'semi': 'error',
+        'no-undef' : 'off'
```

Comment:
typically there are ways to temporarily disable eslint rules in the files themselves (for exceptional cases), maybe that could be done instead? e.g. https://evanhahn.com/disable-eslint-for-a-file/